src/content/docs/alerts-applied-intelligence/new-relic-alerts/alert-conditions/view-alert-conditions.mdx

```diff
@@ -0,0 +1,342 @@
+---
+title: Create and edit alert conditions in the UI
+tags:
+  - Alerts and applied intelligence
+  - Alerts
+  - Alert conditions
+translate:
+  - jp
+metaDescription: "Use the conditions page to identify what triggers an alert policy's notification, starting with the product and type of metric or service."
+redirects:
+  - /docs/alerts-applied-intelligence/new-relic-alerts/get-started/your-first-nrql-condition/
+  - /docs/alerts/alert-policies/configuring-alerts/managing-your-alerts 
+freshnessValidatedDate: never
+---
+import alertsSetaQuery from 'images/alerts_screenshot-crop_set-a-query.webp'
+
+import alertsFineTuneAlertSignals from 'images/alerts_screenshot-crop_fine-tune-alert-signals.webp'
+
+import alertsSetaThresholdforanAlertCondition from 'images/alerts_screenshot-crop_set-a-threshold-for-an-alert-condition.webp'
+
+import alertsNameYourAlertCondition from 'images/alerts_screenshot-crop_name-your-alert-condition.webp'
+
+import alertsAlertConditionPage from 'images/alerts_screenshot-crop_alert-condition-page.webp'
+
+New Relic alerts are built with alert conditions. If you want to be notified of any unusual behavior in your stack, you'll first need to create an alert condition. An alert condition is essentially a container that you create to define what conditions need to be met before you're notified of any unusual behavior. 
+
+<img
+    title="alert condition homepage"
+    alt="A screenshot showing the alert condition homepage"
+    src={alertsAlertConditionPage}
+/>
+<figcaption>
+To use New Relic's universal alert condition experience, go to [one.newrelic.com > All capabilities](https://one.newrelic.com/all-capabilities) > **Alerts & AI**, select **Alert Conditions** in the left navigation.
+</figcaption>
+
+With our **Alert Conditions** page you can:
+* Browse all of your alert conditions 
+* Easily identify any open issues
+* Open existing alert conditions to edit thresholds or signal behavior 
+* Create a new alert condition 
+
+This doc will walk you through how to edit and change [an _existing_ alert condition](/docs/alerts-applied-intelligence/new-relic-alerts/alert-conditions/view-alert-conditions/#edit-existing-alert-condition), and creating [new_ alert condition_](/docs/alerts-applied-intelligence/new-relic-alerts/alert-conditions/view-alert-conditions/#create-alert-condition]). 
```

Comment:
```suggestion
This doc will walk you through how to edit [an _existing_ alert condition](/docs/alerts-applied-intelligence/new-relic-alerts/alert-conditions/view-alert-conditions/#edit-existing-alert-condition) and how to create a [new_ alert condition_](/docs/alerts-applied-intelligence/new-relic-alerts/alert-conditions/view-alert-conditions/#create-alert-condition]). 
```

Reply:
maybe we can further tighten the language with "This doc walks you through how" -- since it's a doc that exists, we can default to present tense